go.mod

```diff
@@ -20,4 +23,5 @@ require (
 	github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2
 	golang.org/x/crypto v0.0.0-20200323165209-0ec3e9974c59
 	golang.org/x/sys v0.0.0-20200327173247-9dae0f8f5775
+	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f // indirect
```

Comment:
(Are all of these something Go does automatically? This PR does not add dependencies to new modules.)